Dockerfile

```diff
@@ -27,41 +27,53 @@ ENV AIIDALAB_DEFAULT_GIT_BRANCH master
 ENV AIIDALAB_DEFAULT_APPS ""
 
 USER root
+WORKDIR /opt/
 
 # Install OS dependencies.
+# libssl-dev needed for ?
```

Comment:
do we know?

Reply:
Not really, those were added in the initial commit: https://github.com/aiidalab/aiidalab-docker-stack/commit/470ba8f681e79cf6da94b26b3291ef34b19d6e25

Reply:
ok, I'll add a comment to that effect but will leave them in for the moment in order not to lead to any unforeseen issues